migcontrol/templates/base.html

```diff
@@ -129,7 +120,7 @@
   {% block before_content %}{% endblock %}
   <div class="container">
   <div class="row">
-    <div class="col-md-8 migcontrol-contents-container">
+    <div class="col-md-12 migcontrol-contents-container">
```

Comment:
This will break all the other pages where it's supposed to be 8 wide. It will push the sidebar below, since it has 4 columns allocated.

Reply:
I think you need to take me through on how you devide